armory/utils/version.py

```diff
@@ -0,0 +1,130 @@
+'''
+ARMORY Versions use "Semantic Version" scheme where stable releases will have versions
+like `0.14.6`.  Armory uses the most recent git tag for versioning. For example if the
+most recent git tag is `v0.14.6`, then the version will be `0.14.6`.
+
+If you are a developer, the version will be constructed from the most recent tag plus a
+suffix of gHASH where HASH is the short hash of the most recent commit. For example,
+if the most recent git tag is v0.14.6 and the most recent commit hash is 1234567 then
+the version will be 0.14.6.g1234567. This scheme does differ from the scm strings
+which also have a commit count and date in them like 1.0.1.dev2+g0c5ffd9.d20220314181920
+which is a bit ungainly.
+'''
+
+import os
+import re
+import site
+import functools
+import setuptools_scm
+
+from pathlib import Path
+
+try:
+    from importlib import metadata
+except ModuleNotFoundError:
```

Comment:
This line should be:
```
except ImportError:
```
The `importlib` package exists, so it won't raise a `ModuleNotFoundError`

Reply:
good find, thanks.